geniza/corpus/models.py

```diff
@@ -621,6 +624,7 @@ def attribution(self):
         # keep track of unique attributions so we can include them all
         extra_attrs_set = set()
         for url in self.iiif_urls():
+            # NOTE: If this url fails, may raise IIIFException
```

Comment:
is attribution only being loaded from remote manifests and not using the cached versions?

Reply:
Looks like it! Good call on using cached manifests where possible. I'll make a new issue.